content/hi/docs/home/_index.md

```diff
@@ -43,22 +43,22 @@ cards:
   title: "प्रशिक्षण"
   description: "कुबेरनेट्स में प्रमाणित हो कर अपने क्लाउड नेटिव प्रोजेक्ट सफल बनाएं!"
   button: "प्रशिक्षण देखें"
-  button_path: "/training"
+  button_path:/docs/reference
```

Comment:
this will redirect to the Reference page in hindi. I dont thing it will redirect to the training pag in hindi.
You can see from the line 51 in the same index.md 
As or refernce page it is using /docs/reference.
WDYT? @dipesh-rawat